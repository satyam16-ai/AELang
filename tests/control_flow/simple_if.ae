// If-else control flow test
extern print(format: str, ...);

func main(): void {
    let x: i32 = 10;
    if (x > 5) {
        print("x is greater than 5\n");
    } else {
        print("x is not greater than 5\n");
    }
}
