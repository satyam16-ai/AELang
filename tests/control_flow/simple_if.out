x is greater than 5
